apollo-common/src/main/resources/application.properties

```diff
@@ -17,3 +17,7 @@ server.max-http-header-size=10240
 management.endpoints.web.base-path=/
 logging.file.max-size=50MB
 logging.file.max-history=10
+
+management.endpoints.web.exposure.include=info,health,metrics,prometheus
+management.endpoint.metrics.enabled=true
```

Comment:
According to [Enabling Endpoints](https://docs.spring.io/spring-boot/docs/current/reference/html/production-ready-endpoints.html#production-ready-endpoints-enabling-endpoints): by default, all endpoints except for shutdown are enabled. So it's not necessary to set metrics and prometheus endpoints enabled?

Reply:
No need indeed, just to show other developers. Or delete it.